docs/how-to/platform-console.md

```diff
@@ -1,99 +1,57 @@
+## Platform Console Features
 
-1. Navigate to the Platform Console dashboard.
-2. Click on "Create Template".
-3. Provide a name for your template (e.g., "Node.js with PostgreSQL").
-4. Select the desired components (e.g., Admin UI, GraphQL API, REST API).
-5. Choose a database (e.g., PostgreSQL).
-6. Click "Create" to finalize the template.
+### Live Service Templates
 
-![Amplication Create New Template](./assets/template-name-and-stack.png)
+Live Service Templates is a powerful feature that allows you to create, manage, and use standardized templates for your backend services. Key aspects include:
 
-### Managing Templates
+- **Template Creation**: Define reusable service blueprints with pre-configured settings and plugins.
+- **Service Generation**: Quickly create new services based on your defined templates.
+- **Template Management**: Edit and update templates, with the ability to propagate changes to services.
 
-Managing and updating templates is very similar to updating individual services. You have access to the same settings and most of the options available for services, including:
-
-- Viewing all templates in the Platform Console dashboard.
-- Adding or removing plugins, including custom plugins.
-- Renaming templates and updating their descriptions.
-
-To edit a template:
-
-1. Navigate to the Settings tab to update the configuration or the Plugins tab to install and update plugins.
-2. Make your desired changes.
-3. All changes are saved automatically.
-
-<!-- :::note
-When you update a template, **all services using that template will be updated in your next build**, ensuring consistency across your infrastructure.
-:::
--->
+For detailed information on creating and using Live Service Templates, please refer to our [Live Service Templates](/live-service-templates) documentation.
 
-### Using Templates
+### Private Plugins
 
-To create a new service based on a template:
-
-1. Click on the "Add Resource" button.
-2. Select "Service from Template" from the available options.
-3. In the modal that appears, choose a specific service template.
-4. Provide a name for your new service.
-5. Click "Create" to generate the new service based on the selected template.
-
-This process ensures that the new service inherits all the pre-configured settings and plugins defined in the template.
-
-![Amplication Create Services From Template](./assets/create-service-from-template.png)
-
-## Private Plugins
-
-Private plugins allow you to create custom functionality specific to your organization's needs. These can be added to templates or directly to services.
+Private plugins allow you to create custom functionality specific to your organization's needs. These can be added to services or used in templates.
 
 :::note
 Creating and managing private plugins requires an Enterprise plan.
 :::
 
-### Creating a Private Plugin
-
 To create a private plugin:
 
 1. In the Platform Console dashboard, navigate to the "Private Plugins" section.
```

Comment:
Let's update the steps here, and link also to the new private plugins' creation page